src/platform/icelake/include/platform/shim.h

```diff
@@ -208,8 +208,8 @@
 #define SHIM_LDOCTL_LPSRAM_MASK	(3 << 2)
 #define SHIM_LDOCTL_HPSRAM_LDO_ON	(3 << 0)
 #define SHIM_LDOCTL_LPSRAM_LDO_ON	(3 << 2)
-#define SHIM_LDOCTL_HPSRAM_LDO_BYPASS	BIT(0)
-#define SHIM_LDOCTL_LPSRAM_LDO_BYPASS	BIT(2)
```

Comment:
Any particular reason to replace BIT macro with regular bit shifting ?

Reply:
`BIT()` does not compile in asm, seems like a limited number of substitutions is made by the pre-processor.